docs/native/src/faq/abortedtokenissuance.rst

```diff
@@ -0,0 +1,4 @@
+Why does calling the Token Issuance URL return "The request was aborted: Could not create SSL/TLS secure channel"?
+==================================================================================================================
+Please configure the NetScaler to just load balance TCP/443 instead of using the built-in SSL protocol load balancing. 
```

Comment:
Seems like you need a little more context here. E.g.

> This can be caused by networking infrastructure that terminates SSL connections, such as <whatever>. Please ensure your networking infrastructure does not <whatever>.

I made that up completely just to illustrate; you'll need to figure out what to add.